File: java/ 3. Calculadora ciclo finito/calcular.java
import java.util.Scanner;
import java.util.InputMismatchException;

public class calcular{
    public static void main(String[] args){
	Scanner Lectura = new Scanner(System.in);
	System.out.println("Programa que calcula la suma, resta, multiplicación y división de dos números");
	try{
	    System.out.println("Ingrese el primer número: ");
	    float num1 = Lectura.nextFloat();
	    System.out.println("ingrese el segundo número");
	    float num2 = Lectura.nextFloat();

	    float Suma;
	    float Resta;
	    float Multi;
	    float Divi;
	    Suma = suma(num1, num2);
	    Resta = resta(num1, num2);
	    Multi = multiplica(num1, num2);
	    Divi = divide(num1, num2);
	    System.out.println("La suma de "+ num1 + " + "+ num2+ " es " + Suma);
	    System.out.println("La resta de "+ num1 + " + "+ num2+ " es " + Resta);
	    System.out.println("La multiplicación de "+ num1 + " + "+ num2+ " es " + Multi);
	    System.out.println("La división de "+ num1 + " + "+ num2+ " es " + Divi);

	}

	catch(InputMismatchException e){
	    System.out.println("No introduzca carecteres, se esperan números.");
	}
	
    }

    static float suma(float a, float b){
	float c;
	c = a+b;
	return c;
    }
    
    static float resta(float a, float b){
	float c;
	c= a-b;
	return c;
    }
    
    static float multiplica(float a, float b){
	float c;
	c = a*b;
	return c;	    
    }
    
    static float divide(float a, float b){
	float c;
	c = a/b;
	return c;
    }
}

File: python/ 3. Calculadora ciclo finito/calcular.py
#*-* coding:utf-8 *-*
def suma(a, b):
    c = a + b
    return c

def resta(a,b):
    c = a - b
    return c

def multiplica(a,b):
    c = a*b
    return c

def divide(a, b):
    c = a/b
    return c

print "Prográma que cálcula la suma, resta, multiplicación y división de dos números."

try:
    a = raw_input("ingrese el primer número: ")
    a = float(a)
    b = raw_input("ingrese el segundo número: ")
    b = float(b)
    try:
        print "La suma de ("+ str(a) + " + " +str(b) + ") es " + str(suma(a,b))
        print "La resta de ("+ str(a) + " - " +str(b) + ") es " + str(resta(a,b))
        print "La multiplicación de ("+ str(a) + " * " +str(b) + ") es " + str(multiplica(a,b))
        print "La división de ("+ str(a) + " / " +str(b) + ") es " + str(divide(a,b))

    except:
        print "La división por cero no eta definida"
except:
    print "No puede ingresar letras, se esperan números."
